trafficFlowTests.py

```diff
@@ -48,6 +48,14 @@ def _cleanup_previous_testspace(self, cfg_descr: ConfigDescriptor) -> None:
             logger.error(r)
             raise Exception("cleanup_previous_testspace(): Failed to delete services")
         logger.info(f"Cleaned services with label tft-tests in namespace {namespace}")
+        logger.info(f"Cleaning multi-networkpolicies with label tft-tests in namespace {namespace}")
+        r = cfg_descr.tc.client_tenant.oc(
+            f"delete multi-networkpolicies -n {namespace} -l tft-tests"
+        )
+        if r.returncode != 0:
+            logger.error(r)
+            raise Exception("cleanup_previous_testspace(): Failed to delete multi-networkpolicies")
```

Comment:
let's not do additional error handling. Just pass `die_on_error=True` to `K8sClient.oc()` to abort on failure.

Or, maybe don't treat such error fatally. `K8sClient.oc()` already does by default `may_fail=False`, which means a failure results in a warning log line. That may be sufficient for as far as handling the error goes.

Reply:
The reason is that the line automatically logged by `K8sClient.oc()` is better than `logger.error(r)`, which lacks any context about what command failed. So, you can do less work (lines of code) while getting the better logging line.

Reply:
@thom311 Sure! I'll make these changes. Also I will rebase my branch. 
I've marked this PR as a draft for now so that I can mark it as ready once I've completed all the changes and tested them.